File: phaseGateOne/StudentGrade.java
import java.util.Scanner;
public class StudentGrade{
	public static void main(String... args){
	Scanner input = new Scanner(System.in);

	int failures = 0;
	int passes = 0;
	int total = 0;
	int average = 0;

	System.out.print("How many students do you have? ");
	int numberOfStudent = input.nextInt();

	System.out.print("How many subjects do they offer? ");
	int numberOfSubjects = input.nextInt();

	System.out.print("Saving >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
	System.out.println("Saved Successfully");
	System.out.println();

	int[][] scoresOfStudents = new int[numberOfStudent][numberOfSubjects];
	for(int index = 1; index <= numberOfStudent; index++){
		for(int number = 1; number < numberOfSubjects; number++){
			System.out.println("Entering score for student " + index);
			System.out.print("Enter score for subject " + number+ " : ");
			scoresOfStudents [index - 1][number -1] = input.nextInt();
			System.out.println("Saving >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
			System.out.println("Saved successfully!");
			System.out.println();
		}
	}
	printScoreBoard(numberOfStudent, numberOfSubjects, scoresOfStudents );
}


public static void printScoreBoard(int numberOfStudent, int numberOfSubjects, int[][] scoresOfStudents){
	System.out.println("====================================================================");
	System.out.print("STUDENT\t\t ");
		for(int counter = 1; counter <= numberOfSubjects; counter++){  
			System.out.print("SUB" +counter+ "\t" );
		}		
	System.out.println("TOT\tAVE\tPOS\t");
	System.out.println("====================================================================");
	int total = 0;
	for(int count = 1; count <= numberOfStudent; count++){
		System.out.print("Student" +count+ "\t");
		for(int countTwo = 1; countTwo <= numberOfSubjects; countTwo++){
			System.out.print(scoresOfStudents[count - 1][countTwo -1 ] + "\t" );
			total += scoresOfStudents[count -1][countTwo -1];		 	}
		
		System.out.print(total + "\t");
		double average = total / numberOfSubjects;
		System.out.printf("%.2f\t", average);
		
		System.out.println("");
	}	
	System.out.println("====================================================================");
	System.out.println();
	System.out.println("====================================================================");




}
//public static int printAverage(int scoreOfStudents){
	//int average = total / numberOfStudent;



//}

//public static String printGradeSummary(){
	//System.out.printf(" The Highest scoring student is: ");
	//System.out.printf(" The Lowest scoring student is: ");
	//System.out.printf(" The Total score is: ");
	//System.out.printf(" The average score is: ");
	//System.out.printf(" The Number of passes: ");
	//System.out.printf(" The Number of fails: ");

//}

//System.out.print(The hardest subject is subject %d, with %d failures" , subject, failures);
//System.out.print(The easiest subject is subject %d, with %d passes" , subject, passes);
//System.out.print(The overall highest Score is scored by %d, in %d scoring %d" , studentId, subject, score);
//System.out.print(The overall lowest score is scored by %d, in %d scoring %d" , studentId, subject, score);

//System.out.println("=============================================================");

//System.out.print("CLASS SUMMARY");
//System.out.println("=============================================================");

//System.out.print("Best Graduating Student is: +student+ scoring +score+");
//System.out.println("==============================================================");

//System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
//System.out.print("Worst Graduating Student is: +student+ scoring +score+");
//System.out.println("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");

//System.out.println("==============================================================");

//System.out.printf("Class total is : %d", total);
//System.out.printf("Class Average score is : %d", average);


	


}

File: phaseGateOne/student_grade.py
	
def get_score_board(number_of_student, number_of_subjects, scores_of_students : list):
print("====================================================================");
print("STUDENT\t\t ")
for counter in number_of_subjects: 
	print(f"SUB" {counter} "\t" )		
	print("TOT\tAVE\tPOS\t\n")
	print("====================================================================")
total = 0;
for count in number_of_student:
	print("Student" {count} "\t")
		for numbers in number_of_subjects:
			print(scores_of_students[count - 1][numbers -1] , "\t")
			total += scores_of_students[count -1][numbers -1]	
		
		print(total , "\t")
		average = total / number_of_subjects
		print(f"%.2f\t" {average});
		println("")
	
	print("====================================================================")
	print()
	print("====================================================================")

def get_input():
failures = 0
passes = 0
average = 0

number_of_student = int(input("How many students do you have? "))

number_of_subjects = int(input("How many subjects do they offer? "))
	print("Saving >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
	println("Saved Successfully.")
	println()

	scores_of_students = [[number_of_student][number_of_subjects]]
	for index in number_of_student:
		for number in number_of_subjects:
			print("Entering score for student " + index)
			print("Enter score for subject " + number+ " : ")
			scores_of_students [index - 1][number -1]
			
		print("Saving >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
		print("Saved successfully.")
		print()
	
	print(printScoreBoard(number_of_student, number_of_subjects, scores_of_students))




//public static void printSubjectSummary() {
	int highestScore =
	int lowestScore = 
	//System.out.printf(" The Highest scoring student is: %d scoring %d");
	//System.out.printf(" The Lowest scoring student is: %d scoring %d");
	//System.out.printf(" The Total score is: %d");
	//System.out.printf(" The average score is: %s");
	//System.out.printf(" The Number of passes: %d");
	//System.out.printf(" The Number of fails: %d");


}

